packages/amplify-provider-awscloudformation/src/aws-utils/aws-s3.ts

```diff
@@ -10,13 +10,15 @@ import { $TSAny, $TSContext, AmplifyError, AmplifyFault, stateManager } from '@a
 
 import _ from 'lodash';
 
+import fs from 'fs-extra';
 import ora from 'ora';
 import { ListObjectVersionsOutput, ListObjectVersionsRequest, ObjectIdentifier } from 'aws-sdk/clients/s3';
 import { pagedAWSCall } from './paged-call';
 import { loadConfiguration } from '../configuration-manager';
 import aws from './aws';
 
 const providerName = require('../constants').ProviderName;
+const consumers = require('stream/consumers');
```

Comment:
I was unable to bend TS compiler to accept any form of `import`, so I switched to `require` to workaround it.

It seems that bumping TS compiler to `5.x` would be required to make it right. (which is out of scope here).
The other alternative would be to not use this Node api and write own consumer which is not ideal either.